lib/pluginmanager/prepare_offline_pack.rb

```diff
@@ -21,7 +22,21 @@ def execute
     # To silence some of debugs/info statements
     Paquet.ui = Paquet::SilentUI unless debug?
 
-    FileUtils.rm_rf(output) if ::File.exist?(output)
+    if File.directory?(output)
+      signal_error("The specified output is a directory, you must specify a filename with a zip extension, provided output: #{output}")
+    else
+      if File.extname(output) !~ /zip/i
```

Comment:
I think we can make this a == instead, otherwise the following is possible:
```
% bin/logstash-plugin prepare-offline-pack --output /tmp/my.maywerzipwer logstash-filter-translate
Offline package created at: /tmp/my.maywerzipwer
```